src/utils/render.js

```diff
@@ -0,0 +1,10 @@
+export function debounce(fn, ms) {
```

Comment:
We already have a dependency on lodash, can we just use their debounce function? https://lodash.com/docs/4.17.15#debounce